src/code/views/download-dialog-view.coffee

```diff
@@ -34,6 +34,10 @@ module.exports = React.createClass
     s.replace /^\s+|\s+$/, ''
 
   download: (e) ->
+    makeBlobURL = (msg) ->
+      blob = new Blob([msg], {type: 'text/plain'})
+      window.URL.createObjectURL(blob)
```

Comment:
Made the suggested change (BTW, the site caniuse.org is also pretty good for compatibility issues. e.g.:  http://caniuse.com/#feat=bloburls)